pages/index.vue

```diff
@@ -184,4 +188,7 @@ const uploadHandler = async (e: any) => {
   }
   uploading.value = false;
 };
+
+import createVueFilePond from "vue-filepond";
+const FilePond = createVueFilePond();
```

Comment:
when deployed to vercel, it complains that "createVueFilePond is not a function", and throws an 500 error.